alacritty.yml

```diff
@@ -178,6 +178,13 @@
   # it is recommended to set `use_thin_strokes` to `false`.
   #use_thin_strokes: true
 
+  # Use built-in font for box drawing characters.
+  #
+  # If `true`, Alacritty will use a custom built-in font for box drawing
+  # characters (Unicode points 2500 - 259f).
+  #
+  #use_built_in_font_for_box_drawing: false
```

Comment:
Could we default it to `true`? I'd also name it `builtin_box_drawing`, since we're in `font` section already. The reason behind true, since most terminals I'm aware of has it to true, and it makes sense.

Reply:
That's fine. Since this is already in a release it would be a breaking change.

Other than changing this to `true` here in the default config, is there a way to specify the default in the code?

Reply:
You need to modify `Default` impl for `Font`. You could try to search for `impl Default` in config code to look at examples.